src/wagtailbakery/api_views.py

```diff
@@ -1,16 +1,20 @@
+import os
 import json
 import logging
-import os
 
-from bakery.views import BuildableMixin
 from django.conf import settings
-from django.contrib.contenttypes.models import ContentType
-from wagtail.api.v2.endpoints import PagesAPIEndpoint
-from wagtail.api.v2.router import WagtailAPIRouter
+from bakery.views import BuildableMixin
+
 from wagtail.core.models import Page, Site
+from wagtail.api.v2.router import WagtailAPIRouter
+from wagtail.api.v2.endpoints import PagesAPIEndpoint
+
+from django.contrib.contenttypes.models import ContentType
```

Comment:
the import order should not change - the isort test fails now - https://travis-ci.org/wagtail/wagtail-bakery/jobs/631872784

Reply:
will revert but i tend to like neat imports